src/omv/common/sensor.h

```diff
@@ -38,12 +38,12 @@
 #define OV7725_ID           (0x77)
 #define OV9650_ID           (0x96)
 #define MT9V034_ID          (0x13)
-#define MT9M114_ID          (0x81)
```

Comment:
Let's keep it here to match the order of init.

Reply:
Fixed